verilog/analysis/verilog_linter.cc

```diff
@@ -332,10 +333,51 @@ absl::Status PrintRuleInfo(std::ostream* os,
   return absl::OkStatus();
 }
 
+static void AppendCitation(CustomCitationMap& citations,
+                           const absl::string_view& rule_raw) {
+  const size_t eq_pos = rule_raw.find(':');
```

Comment:
`eq_pos` sounds like it was re-used from some other code where we have an equal.... here this should probably be more `colon_pos`.

Reply:
At first, I decided to use the equal sign, but in the end, I changed my mind to colon and forgot about changing the variable names.